### PR TITLE
update `PackageInfo` and `ProjectInfo` docs

### PR DESCRIPTION
**Pull request by t-bltg**
Some field descriptions were missing.

### CODE REVIEW

**Comment by t-bltg** [comment]
@IanButterworth, can this harmless PR be merged ?

File: src/Pkg.jl [hunk 1]
```diff
@@ -364,15 +364,19 @@ The result is a `Dict` that maps a package UUID to a `PackageInfo` struct repres
...
-| `version`         | The version of the package (this is `Nothing` for stdlibs) |
-| `is_direct_dep`   | The package is a direct dependency                         |
-| `is_tracking_path`| Whether a package is directly tracking a directory         |
-| `is_pinned`       | Whether a package is pinned                                |
-| `source`          | The directory containing the source code for that package  |
-| `dependencies`    | The dependencies of that package as a vector of UUIDs      |
+| Field                 | Description                                                                       |
+|:------------------    |:----------------------------------------------------------------------------------|
+| `name`                | The name of the package                                                           |
+| `version`             | The version of the package (this is `Nothing` for stdlibs)                        |
```
Comment (IanButterworth) [inline src/Pkg.jl:370]:
```suggestion
| `version`             | The version of the package (this is `Nothing` for stdlibs)                        |
| `tree_hash`           | A file hash of the package directory tree                                         |
```

**Review by IanButterworth — CHANGES_REQUESTED** [review]

File: src/Pkg.jl (hunk 1 above)
Comment (IanButterworth) [reply src/Pkg.jl:370]: Or was this omitted for a reason?

Comment (t-bltg) [reply src/Pkg.jl:370]: I think I just missed it.

**Review by IanButterworth — APPROVED** [review]

**Pull request merged**